docs/element-definitions.md

```diff
@@ -773,22 +811,22 @@ Type Element matching only a property list with key "foo" of value type `boolean
 
 ### Enum Element
 
-Type with the domain of any of the types listed in the `enumerations` attribute. Also called tagged union or Σ-type.
+Type with domain of the union of values typed by Elements in the `enumerations` attribute. Also called tagged union or Σ-type.
 
 #### Template
 - `element` - `"enum"`
 - `attributes`
-  - `enumerations` ([Array](#array-element)) - list of Elements which this Element SHALL type
+  - `enumerations` ([Array](#array-element)) - List of Elements
   - `typeAttributes` ([Array](#array-element)[[String](#string-element)])
     - `fixed` ([String](#string-element)) - Elements in `enumerations` SHALL be interpreted `fixed`.
   - `validation` - _reserved for future use_
-  - `samples` ([Array](#array-element)) - array of alternative sample values for this Element; type of items in `samples` MUST match the type of this Element
-  - `default` (Element) - default value for this Element; type of `default` MUST match the type of this Element
-- `content` - an Element matching one of the Elements in the `enumerations` attribute
+  - `samples` ([Array[[Element](#types)]](#array-element)) - Alternative sample values for this Element; type of items in `samples` MUST match the type this Element describes
```

Comment:
Bad linking. Use something similar to `([Array](#array-element)[[String](#string-element)])`